opentelemetry-sdk/src/opentelemetry/sdk/metrics/view.py

```diff
@@ -68,36 +68,64 @@ class View:
     def __init__(
         self,
         metric: InstrumentT,
-        aggregator: Aggregator,
+        aggregator: type,
+        aggregator_config: dict = None,
         label_keys: Sequence[str] = None,
-        config: ViewConfig = ViewConfig.UNGROUPED,
+        view_config: ViewConfig = ViewConfig.UNGROUPED,
     ):
         self.metric = metric
         self.aggregator = aggregator
+        if aggregator_config is None:
+            aggregator_config = {}
+        self.aggregator_config = aggregator_config
         if label_keys is None:
             label_keys = []
         self.label_keys = sorted(label_keys)
-        self.config = config
+        self.view_config = view_config
+        self.view_datas = set()
+
+    def get_view_data(self, labels):
+        active_labels = []
+        if self.view_config == ViewConfig.LABEL_KEYS:
+            active_labels = {(lk, lv) for lk, lv in labels if lk in set(self.label_keys)}
```

Comment:
Nice. I can't really think of a nicer way to do this. It would be great if you could explain what this step is for (finding instances of previously created `viewdata` based off currently passed in labels.

Reply:
done